pages/linux/top.md

```diff
@@ -18,6 +18,10 @@
 
 `top -p $(pgrep -d ',' {{process_name}})`
 
+- Sort by memory usage with memory units (k/M/G):
+
+`top -aM`
```

Comment:
Just tested this and I seem to get an error of `top: unknown option 'a'`. Is this a standard command?

Reply:
Sorry! As you described, it's not standard.

It works on:

$ cat /etc/redhat-release
CentOS release 6.9 (Final)

So, as it's not standard, I think it's not a good idea.
Apologies for any inconvenience.

Reply:
No problem. 👍

I'll leave this here to see if it should be included anyway.
@sbrl @agnivade

Reply:
Probably not a good idea.

Reply:
Hrm. What version of `top` are you using? If I do `top -v`, I get this:

```
  procps-ng 3.3.12
Usage:
  top -hv | -bcHiOSs -d secs -n max -u|U user -p pid(s) -o field -w [cols]
```

....I'm just wondering if it's a similar case to `netcat` - i.e. there's both a BSD `netcat` and a GNU one.

Reply:
I see the same here for centOS - https://www.unix.com/man-page/centos/1/top/. But this does not contain the `-aM` option.

Reply:
I was using 3.3.12 on Ubuntu 17.10.

I've also tested on my NAS and CentOS server. 👍